doc/reference/settings/index.rst

```diff
@@ -139,6 +139,14 @@ When storage becomes full (FCB) or consumes too much space (file system),
 the backend removes non-recent key-value pairs records and unnecessary
 key-delete records.
 
+Secure domain settings
+**********************
+Currently setting doesn't provide scheme of being secure, and non-secure
```

Comment:
settings*

Reply:
This sentence is a little bit confuse for me, but I'm a native speaker.

Reply:
I'm not - you can suggest somethin which will sounds better.